docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,600 @@
+YODA-compliant data analysis projects
+-------------------------------------
+
+Now that you know about the YODA principles and DataLad's Python API, it is
+time to start working on ``DataLad-101``'s midterm project.
+In order to combine the previous sections, this midterm project is a
+YODA-compliant data analysis project, written in Python, and set up in a
+DataLad dataset.
+
+For your submission, you decide to analyze the
+`iris flower data set <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_.
+It is a multivariate data set on 50 samples of each of three species of Iris
+flowers (*Setosa*, *Versicolor*, or *Virginica*), with four variables: the length and width of the sepals and petals
+of the flowers in centimeters. It is often used in introductory data science
+courses for statistical classification techniques in machine learning, and
+widely available, among many other sources as
+`GitHub gists <https://gist.github.com/netj/8836201>`_.
+
+Raw data as a modular, independent entity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To start your analysis project and comply to the YODA principles, you set up
+an independent data dataset with your project's raw data. For this, create a
+new dataset outside of ``DataLad-101``:
```

Comment:
I am still torn between the "create yourself a dataset" approach that you are taking here vs. us providing a dataset for this at https://github.com/datalad-handbook

Your approach is more self-contained, but cutting the prelude in favor of a plain `install` call removes some distractions and orthogonalizes the content with other chapters.

Reply:
I'm indifferent - I'll push a dataset to https://github.com/datalad-handbook that can be installed as input and maybe do a hidden section about "if I have collected my own data..."